hs_script_resource/models.py

```diff
@@ -53,7 +53,7 @@ class ScriptSpecificMetadata(AbstractMetaDataElement):
                                              help_text='The date that this version of the script was released')
 
     # repository
-    scriptCodeRepository = models.CharField(verbose_name='Script Repository', blank=True, max_length=255,
+    scriptCodeRepository = models.URLField(verbose_name='Script Repository', blank=True, max_length=255,
```

Comment:
I went ahead and made this change. It doesn't look like this field is being used actively. What do you think @aphelionz ?